File: egypt/template/egypt.java
import java.util.*;

public class egypt {

    // return True if a, b, and c form a right triangle
    // return False otherwise
    public static boolean isRightTriangle(int a, int b, int c) {
        return false;
    }

    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        int a, b, c;
        do {
            a = in.nextInt();
            b = in.nextInt();
            c = in.nextInt();
            System.out.println(isRightTriangle(a, b, c) ? "right" : "wrong");
        } while (a + b + c != 0);
        in.close();
    }

}

File: egypt/template/egypt.py
# a, b, and c are positive integers
# return True if they form a right triangle
# return False otherwise
def is_right_triangle(a: int, b: int, c: int) -> bool:
    return False

if __name__ == '__main__':
    a, b, c = map(int, input().split())
    while a + b + c:
        print('right' if is_right_triangle(a, b, c) else 'wrong')
        a, b, c = map(int, input().split())